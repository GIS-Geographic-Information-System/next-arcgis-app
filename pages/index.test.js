import React from 'react';
import { render } from 'react-testing-library';
import Index from './index';

describe('smoke tests', function() {
  describe('w/o a previous session', function() {
    const title = 'Test Title';
    it('renders title', () => {
      const { getByText } = render(<Index title={title} />);
      expect(getByText(title)).toBeInTheDocument();
    });
    it('matches snapshot', () => {
      const { asFragment } = render(<Index title={title} />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
