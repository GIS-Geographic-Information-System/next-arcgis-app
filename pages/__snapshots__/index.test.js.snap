// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`smoke tests w/o a previous session matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="jumbotron"
  >
    <h1
      class="display-3 text-light text-center mb-5"
    >
      Test Title
    </h1>
    <form
      class="search-form undefined"
    >
      <div
        class="input-group input-group-lg"
      >
        <input
          class="form-control"
          placeholder="search for items"
          value=""
        />
        <div
          class="input-group-append"
        >
          <button
            class="btn btn-secondary"
            type="submit"
          >
            Search
          </button>
        </div>
      </div>
    </form>
  </div>
</DocumentFragment>
`;
